src/main/osd/osd_elements.c

```diff
@@ -986,6 +986,8 @@ static void osdElementFlymode(osdElementParms_t *element)
         strcpy(element->buff, "HEAD");
     } else if (FLIGHT_MODE(ANGLE_MODE)) {
         strcpy(element->buff, "ANGL");
+    } else if (FLIGHT_MODE(ALTHOLD_MODE)) {
+        strcpy(element->buff, "ALTHOLD");
```

Comment:
is 6 characters safe? It may break UI

Reply:
It shouldn't. There is 32 byte buffer for osd elements (OSD_ELEMENT_BUFFER_LENGTH).

Reply:
And other OSD element overwrite?

Reply:
User place OSD elements manually, he can overwrite them even with 4-letter modes.
I don't like `ALT` or `ALTH` abbreviation, it's not very clear.